src/test/java/it/mulders/mcs/search/printer/DelegatingOutputPrinterTest.java

```diff
@@ -1,27 +1,24 @@
-package it.mulders.mcs.search;
+package it.mulders.mcs.search.printer;
 
+import it.mulders.mcs.search.SearchQuery;
+import it.mulders.mcs.search.SearchResponse;
 import org.apache.commons.io.output.NullOutputStream;
 import org.assertj.core.api.WithAssertions;
 import org.junit.jupiter.api.DisplayNameGeneration;
 import org.junit.jupiter.api.DisplayNameGenerator;
 import org.junit.jupiter.api.Test;
 
-import java.io.OutputStream;
 import java.io.PrintStream;
 
-import static org.mockito.Mockito.any;
-import static org.mockito.Mockito.eq;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.never;
-import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.*;
```

Comment:
Could you please revert the wildcard imports here?

Reply:
Wildcard imports reverted.